CMakeLists.txt

```diff
@@ -19,6 +19,8 @@ cmake_minimum_required(VERSION 3.17.2)
 
 project(VEL LANGUAGES CXX)
 
+option(BUILD_WERROR "Treat compiler warnings as errors")
```

Comment:
Revert the change to BUILD_WERROR. It will be treated as false if it isn't defined. Which works out since the default value is OFF.

This same logic is used for BUILD_TESTS. See scripts/CMakeLists.txt

Reply:
Maybe but I guess / hope it's going to change.